opendatahub-io/odh-dashboard
# Add accelerator table

Closes: #1373

## Description
Added Accelerator profile table
- [X] Render the following columns: Device identifier, display name/description, ennoblement switch, date last modified, action button
- [X] Columns (except for action column) must be sortable
- [X] include help icons where present in the docs
- [X] should include pagination
- [X] ellipsis action button should include: edit, delete
- [X] hook to fetch accelerator CRDs

![image](https://github.com/opendatahub-io/odh-dashboard/assets/97534722/b1f1b6e3-8e53-4790-844e-f7064de4e4cf)

## How Has This Been Tested?
1. Add accelerator profile CR's to your cluster.
2. Login to ODH dashboard with clusteradmin
3. Navigate to settings < Accelerator profiles
4. You can see the Accelerator table, with all it's fields, sortable columns, and action buttons.

## Test Impact
Added storybook component and integration test.

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:

@yannnz
